.cargo/config.toml

```diff
@@ -1,5 +1,7 @@
 [alias]
 xtask = "run --package xtask --"
+metrics = "run --package xtask -- metrics"
+metrics-sql = "run --package xtask --features=sql -- metrics"
```

Comment:
I'm assuming sqlx's offline mode didn't work :P?

Reply:
No, the documentation seems to be wrong, they mention a .sqlx which did not get generated, and I thought this works for now, and and we could play with the offline mode later.
Also having sqlx as a dependency requires openssl to be installed on the image, which we didn't have.